node/tests/TestUtilities.ts

```diff
@@ -692,43 +704,108 @@ export async function DumpAndRestureTest(
-    const key3 = "{key}" + uuidv4(); // string
-    const key4 = "{key}" + uuidv4(); // hash
-    const key5 = "{key}" + uuidv4();
-    const key6 = "{key}" + uuidv4();
-    const key7 = "{key}" + uuidv4();
-    const key8 = "{key}" + uuidv4();
-    const key9 = "{key}" + uuidv4();
-    const key10 = "{key}" + uuidv4();
-    const key11 = "{key}" + uuidv4(); // hyper log log
-    const key12 = "{key}" + uuidv4();
-    const key13 = "{key}" + uuidv4();
-    const key14 = "{key}" + uuidv4(); // sorted set
-    const key15 = "{key}" + uuidv4(); // list
-    const key16 = "{key}" + uuidv4(); // list
-    const key17 = "{key}" + uuidv4(); // bitmap
-    const key18 = "{key}" + uuidv4(); // Geospatial Data/ZSET
-    const key19 = "{key}" + uuidv4(); // bitmap
-    const key20 = "{key}" + uuidv4(); // list
-    const key21 = "{key}" + uuidv4(); // list for sort
-    const key22 = "{key}" + uuidv4(); // list for sort
-    const key23 = "{key}" + uuidv4(); // zset random
-    const key24 = "{key}" + uuidv4(); // list value
-    const key25 = "{key}" + uuidv4(); // Geospatial Data/ZSET
-    const key26 = "{key}" + uuidv4(); // sorted set
-    const key27 = "{key}" + uuidv4(); // sorted set
-    const field = uuidv4();
-    const value = uuidv4();
-    const groupName1 = uuidv4();
-    const groupName2 = uuidv4();
-    const consumer = uuidv4();
+    // initialize key values to work within the same hashslot
+    const [
+        key1, // string
+        key2, // string
+        key3, // string
+        key4, // hash
+        key5, // list
+        key6, // list
+        key7, // set
+        key8, // sorted set
+        key9, // stream
+        key10, // string
+        key11, // hyper log log
+        key12, // sorted set
+        key13, // sorted set
+        key14, // sorted set
+        key15, // list
+        key16, // list
+        key17, // bitmap
+        key18, // Geospatial Data/ZSET
+        key19, // bitmap
+        key20, // list
+        key21, // list for sort
+        key22, // list for sort
+        key23, // zset random
+        key24, // list value
+        key25, // Geospatial Data/ZSET
+        key26, // sorted set
+        key27, // sorted set
+    ] = Array.from({ length: 27 }, () =>
+        decodeString("{key}" + uuidv4(), decoder),
+    );
+
+    // initialize non-key values
+    const [value, groupName1, groupName2, consumer] = Array.from(
+        { length: 4 },
+        () => decodeString(uuidv4(), decoder),
+    );
+
+    const fieldStr = uuidv4();
+    const [
+        field,
+        field1,
+        field2,
+        field3,
+        field4,
+        value1,
+        value2,
+        value3,
+        foo,
+        bar,
+        baz,
+        test,
+        one,
+        two,
+        three,
+        underscore,
+        non_existing_member,
+        member1,
+        member2,
+        member3,
+        member4,
+        member5,
+        member6,
+        member7,
+        palermo,
+        catania,
+    ] = [
```

Comment:
What is the benefit of that? It just takes twice more locs

Reply:
because of the formatter... but it's fewer characters of code ;)